packages/modular-scripts/src/utils/getAllWorkspaces.ts

```diff
@@ -12,14 +13,22 @@ type YarnWorkspaces = Record<string, YarnWorkspace>;
 
 function _getAllWorkspaces(): YarnWorkspaces {
   const modularRoot = getModularRoot();
-  return JSON.parse(
+
+  // strip out ANSI color codes and escape characters
+  const strippedStd = stripAnsi(
     execa.sync('yarnpkg', ['workspaces', 'info'], {
```

Comment:
If you run `yarnpkg --silent workspaces info` then it won't add the ansi.

Reply:
--silent will remove the command and `Done in` but not the ansi characters (in the debugger).  I'll add this in!